tests/js/spec/views/dashboardsV2/issueWidgetCard.spec.tsx

```diff
@@ -160,7 +161,7 @@ describe('Dashboards > IssueWidgetCard', function () {
 
     await tick();
```

Comment:
since we are touching this file. Do you think we can replace the await tick with

`await screen.findByLabeltext`?

Reply:
ahh just learned about `findBy` vs `getBy` 😄 yes that's a great point!